src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -6136,6 +6198,9 @@ void CalcHeatBalanceOutsideSurf(EnergyPlusData &state,
 
     bool MovInsulErrorFlag = false; // Movable Insulation error flag
 
+    // set ground surfaces average temperature
+    GetGroundSurfacesTemperatureAverage(state);
+
```

Comment:
This function calculates ground surfaces average temperature. The ground surfaces average temperature is view factors weighted ground surfaces temperature for use with exterior surface LWR exchange.